### PR TITLE
Submit CC Likert_scale.Rmd for CC group 2

### PR DESCRIPTION
**Pull request by odileeyasu**
add community contribution for cc group 2: Jingyi An, Tingyi Lu

### CODE REVIEW

File: Likert_Scale.Rmd [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+Which of the following is a well-designed Likert scale? Why or why not?
+
+A. Not Satisfied, Somehow Satisfied, Very Satisfied
+
+B. Do more harm than good, Make no difference, Do more good than harm
+
+C. No Impact, Moderate Impact, Huge Impact
+
+
+## 3. Visualization of a Sample Likert Scale Data: Explore Climate Change in the American Mind
```
Comment (jtr13) [inline Likert_Scale.Rmd]: Remove 3.

File: Likert_Scale.Rmd [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+```
+
+```{r stacked bar with refuse removed}
+# ggplot2 stacked bar chart with refused removed
+
+CCAM_rmrf <- CCAM_tidy[CCAM_tidy$Response!="Refused",]
+CCAM_rmrf_summary <- CCAM_rmrf %>%
+  group_by(Question, Response) %>%
+  summarize(Freq = n()) %>%
+  mutate(prop = Freq/sum(Freq)) %>%
```
Comment (jtr13) [inline Likert_Scale.Rmd]: remove extra %>% at end of line

**Comment by jtr13** [comment]
Fantastic chapter! Please make the following small changes and rename the Rmd using only lower case letters. Then it will be ready to be merged.

**Comment by odileeyasu** [comment]
Hi Professor Robbins, I have made the changes you mentioned.

**Comment by jtr13** [comment]
Looks good, thanks for your contribution! 🎉 
Please add the link of the published chapter to your community contribution GitHub assignment.

**Pull request merged**